papers/vyas_ramasubramani/94_vyas_ramasubramani.rst

```diff
@@ -348,28 +341,33 @@ As a result, our initial comparisons will focus on existing packages that solve
 Of these tools, some of the best known are Fireworks :cite:`Fireworks`, AiiDA :cite:`Pizzi2016`, Sacred :cite:`sacred`, and Sumatra :cite:`sumatra`.
 Fireworks and AiiDA are full-featured workflow managers that, like ``signac-flow``, interface with high performance compute clusters to execute complex, potentially nonlinear workflows.
 These tools in fact currently offer more powerful features than ``signac-flow`` for monitoring the progress of jobs, features that are supported by the use of databases on the back end.
-However, accessing data generated by these tools outside the context of the workflow definition is more challenging than it would be with ``signac`` because the data representation is closely tied to the workflows.
-Additionally, the need to maintain a server for workflow management can be cumbersome and introduces additional unnecessary complexities.
-
-.. TODO To validate this comparison, it would be important to know how the "signac database" works and how it is implemented to provide MongoDB-like querying.
-
-Sacred and Sumatra are slightly different tools in that their primary focus is maintaining data provenance, not orchestrating complex workflows.
-Superficially, the ``signac`` framework appears quite similar to Sacred.
-Both use decorators to convert functions into command line executable operations, and configurations can be injected into these functions (in ``signac``'s case, using the job object).
-However, the similarities largely stop there.
-The focus of Sacred is to track provenance by recording information such as when an operation is executed, the configuration used, and whether any data was saved.
+However, maintaining a server for workflow management can be cumbersome, and it introduces additional unnecessary complexities.
+
+A more significant limitation of these other tools is that their data representations are closely tied to the workflow execution, making it much more challenging to access the data outside the context of the workflow.
```

Comment:
That is not true for sacred or sumatra, since they don't technically manage data. I would state this a bit more carefully:
> A more significant limitation of many other tools ... is the tight coupling of workflow model and data representation